lib/redirect_static_pages.rb

```diff
@@ -0,0 +1,48 @@
+# This enables us to redirect static pages published by PublishStaticPages
+# to specified URLs, and remove them from rummager.
+
+class RedirectStaticPages
+  def redirect
+    index = Whitehall::SearchIndex.for(:government)
+    pages.each do |page|
+      Whitehall.search_client.delete_content(page[:base_path])
+      payload = present_for_publishing_api(page)
+      Services.publishing_api.put_content(payload[:content_id], payload[:content])
+      Services.publishing_api.publish(payload[:content_id], nil, locale: "en")
+    end
+  end
+
+  def pages
+    [
+      {
+        base_path: "/government/announcements",
+        content_id: "88936763-df8a-441f-8b96-9ea0dc0758a1",
+        target_path: "/redirect/announcements",
+        locales: Locale.non_english.map(&:code),
+      },
+    ]
+  end
+
+private
+
+  def present_for_publishing_api(page)
+    {
+      content_id: page[:content_id],
+      content: {
+        base_path: page[:base_path],
+        document_type: "redirect",
```

Comment:
Explicitly sending a `redirect` to the Publishing API was somewhat superseded by the `unpublish` action, would it be possible to unpublish rather than publish a redirect here?

Reply:
@cbaines you are right. Thanks a lot, I wasn't aware of this. I amended the commit.

Reply:
Rummager has two different ways of updating it's database, I believe the norm for Whitehall content is for Whitehall to directly talk to Rummager to insert or delete things. Maybe call `SearchIndex.delete` [1]?
1: https://github.com/alphagov/whitehall/blob/master/lib/whitehall/search_index.rb#L16-L18

Reply:
Finders are [migrated](https://github.com/alphagov/rummager/blob/c4ee441b83d252f41531cda4abb0b31fc347b466/config/govuk_index/migrated_formats.yaml#L48) so direct communication with Rummager shouldn't be necessary

Reply:
Fantastic, thank for you for the context @cbaines @sihugh! I've removed the call to rummager from the commit.